soccer/Robot.cpp

```diff
@@ -235,6 +236,10 @@ const Geometry2d::Segment OurRobot::kickerBar() const {
     return Segment(pose * L, pose * R);
 }
 
+Geometry2d::Point OurRobot::mouthPos() const {
```

Comment:
It's a ton of boost python voodo magic in robocup-py. Here's the place you would add it (follow the general structure for adding methods there).

https://github.com/RoboJackets/robocup-software/blob/master/soccer/gameplay/robocup-py.cpp#L745-L781

If it dosen't work, don't sweat it, and I'll add it later (probably after comp though), just remind me :smile: